swarm/api/routes/autopilot_routes.py

```
@@ -0,0 +1,470 @@
+            wisdom_apply_result=wisdom_result,
+        )
+
+    except ValueError as e:
+        raise HTTPException(
+            status_code=404,
+            detail={
+                "error": "autopilot_not_found",
+                "message": str(e),
+                "details": {"run_id": run_id},
+            },
+        )
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)